### DISCUSSION TITLE
Fusionboard Case  mit 2.42 display als stl.datei

### DISCUSSION THREAD

**Discussion by Harisjems** [hoylabs/OpenDTU-OnBattery#1332] [General]
Hallo zusammen, hat jemand zufällig eine passende Fusion Board Case mit 2,42-Zoll-Display als STL-Datei, die er zur Verfügung stellen könnte?
Bei Printables oder 3printd-Solution konnte ich leider keine finden.

**Comment by bovcan** [comment]
it look not

**Comment by dirksteiner** [comment]
> Hallo zusammen, hat jemand zufällig eine passende Fusion Board Case mit 2,42-Zoll-Display als STL-Datei, die er zur Verfügung stellen könnte? Bei Printables oder 3printd-Solution konnte ich leider keine finden.

Ist zwar schon eine Weile her, dass die Frage gestellt wurde, aber ich habe nur vor einigen Tagen bei Thingverse was gesucht. 

https://www.thingiverse.com/thing:6924241

**Reply by The-Holgi** [reply]
Hallo, habe für das Gehäuse einen passenden abgewinkelten Sockel erstellt, so das das Display besser abzulesen ist: https://www.thingiverse.com/thing:6957536
Vielleicht kann das ja jemand gebrauchen.

**Comment by Harisjems** [comment]
Danke, ich habe zwischenzeitlich andere Lösungen mit Remote-Display von @ zurrmaxe gewählt. Fusion tut ihren Job auch ohne Display.
